internals/pasture/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+if(ENABLE_psi4pasture)
+  ExternalProject_Add(psi4pasture
+    DEPENDS psi4_core
+    GIT_REPOSITORY https://github.com/psi4/psi4pasture
+    CMAKE_ARGS -DENABLE_ccsort=${ENABLE_ccsort}
+               -DENABLE_transqt2=${ENABLE_transqt2}
+               -Dpsi4_DIR=${STAGED_INSTALL_PREFIX}/share/cmake/psi4
+               -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+               -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+               -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+    CMAKE_CACHE_ARGS -DCMAKE_C_FLAGS:STRING=${CMAKE_C_FLAGS}
+                     -DCMAKE_CXX_FLAGS:STRING=${CMAKE_CXX_FLAGS}
+    INSTALL_COMMAND ${CMAKE_MAKE_PROGRAM} install)
```

Comment:
Also, when pasture is built at the same time as psi4:
Pasture's install location ends up being in the `stage` directory. However since it is installed before here before the larger project is copied from `stage` -> `final location`, when psi4 is installed the `stage/…../driver/pasture` directory gets copied to the final install location as well.